src/widgets/category/list/item/item-view.js

```diff
@@ -47,7 +47,8 @@ module.exports = cdb.core.View.extend({
 
   _initBinds: function () {
     this.model.bind('change', this.render, this);
-    this.widgetModel.bind('change:search change:isColorsApplied change:prefix change:suffix', this.render, this);
+    this.widgetModel.bind('change:search change:isAutoStyle change:prefix change:suffix', this.render, this);
+    this.dataviewModel.bind('change:autoStyle', this.render, this);
```

Comment:
Do we need to listen to both the `change:isAutoStyle` event on the widgetModel and the `change:autoStyle` event on the dataviewModel?